backend/inspirehep/editor/authorlist_utils.py

```diff
@@ -8,9 +8,13 @@
     r"(\\255|\u02D7|\u0335|\u0336|\u2212|\u2013|\u002D|\uFE63|\uFF0D)", re.UNICODE
 )
 re_multiple_space = re.compile(r"\s{2,}", re.UNICODE)
+re_multiple_comma = re.compile(r", ,", re.UNICODE)
```

Comment:
I wanted to clean the text a bit. Since many times there are ```,  ,``` that is unnecessary, one , is enough.
Example, when there are many , in the text after replacing chars:
```'A. Aduszkiewicz ,  1 ,   ,   Y.X. Ali ,  1 ,  20 ,   ,   E I Andronov ,  20 ,   ,   Einstein ,  13 ,  15'```
Would this ```re.compile(r"\s,\s,\s", re.UNICODE)``` be better?

Reply:
maybe if that's the case allow in regex zero or multiple spaces between commas, maybe text might come also in that form if indeed it happens